modules/local/sanntis.nf

```diff
@@ -15,14 +15,16 @@ process SANNTIS {
     path "versions.yml"                   , emit: versions
 
     script:
+    def genbank_extension = prokka_gbk.extension
     if (interproscan_tsv.extension == "gz") {
         """
         gunzip -c ${interproscan_tsv} > interproscan.tsv
+        grep -v "/protein_id=" ${prokka_gbk} > ${meta.prefix}_prepped.${genbank_extension}
```

Comment:
This is because `SanntiS` doesn't like that bit on the gbk file?

Reply:
Basically yes - it preferentially takes the protein name from the protein_id field but in Bakta the format of it is different and causes SanntiS to output an empty file. With that line removed, it uses the locus tag to get the name and that works well. I reported it but using this as a temporary solution.